package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "webpack-howto-example",
+  "version": "1.0.0",
+  "description": "Skeleton React app using webpack, less, JSX, ES6, and react-router.",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack-dev-server",
+    "test": "echo 'Error: no test specified' && exit 1"
+  },
+  "author": "https://github.com/petehunt",
+  "license": "ISC",
+  "devDependencies": {
+    "babel-loader": "*",
```

Comment:
These should be locked down and a npm-shrinkrwrap.json file should be created for a real app.